.babelrc

```diff
@@ -6,5 +6,33 @@
   ],
   "plugins": [
     "lodash"
-  ]
+  ],
+  "env": {
+    "development": {
```

Comment:
Moved all babel config into `babelrc`.  The `development` section was previously in the loader query.